gitgud/test_operator.py

```diff
@@ -64,3 +72,34 @@ def test_mark_level_hierarchy(file_operator, level):
             initial_status = level.get_progress()
             file_operator.mark_level(level, weaker)
             assert initial_status == level.get_progress()
+
+
+def test_file_in_commit(file_operator, gg, content_level):
+    gg.load_level(content_level)
+    assert not file_operator.file_in_commit("master", "welcome.txt")
+    assert file_operator.file_in_commit("master", "Welcome.txt")
+
+
+def test_get_commit_content(file_operator, gg, content_level):
+    gg.load_level(content_level)
+    assert "Welcome" in file_operator.get_commit_content("HEAD", "Welcome.txt")
```

Comment:
Add another test. If we're actually caching the result of this, we should check that it's in the dictionary, so we should check that the function returns the right result.

We should ideally also check to see if this method works with commits, and works when you call it once and call it again with a different file. 

To make this clear:
```
assert "Welcome" in op.gcc("HEAD", "Welcome.txt")
assert "Welcome" in op.gcc("HEAD", "Welcome.txt") # Since it's now cached
assert "Hello" in op.gcc("HEAD", "Hello.txt")
assert "Hello" in op.gcc(op.repo.head.branch, "Hello.txt")
assert "Hello" in op.gcc(op.repo.head.commit.hexsha, "Hello.txt")
```
There may be typos in that